packages/grafana-data/src/types/select.ts

```diff
@@ -6,6 +6,7 @@ export interface SelectableValue<T = any> {
   value?: T;
   imgUrl?: string;
   icon?: string;
+  iconPlacement?: 'left' | 'right';
```

Comment:
One last nit: Since the icon placement seems very RadioButton specifit ATM, I think I would go with a RadioButton specific interface extension.

So, instead of RadioButtonGroup accepting 

```
options: Array<SelectableValue<T>>
```

I suggest:
```
interface RadioButtonSelectableValue<T> extends SelectableValue<T> {
  iconPlacement?: 'left' | 'right'
}

...
options: Array<RadioButtonSelectableValue<T>>
```

This is a cosmetic change, but think it's better to scale it later for all selectable values rather than polutting public interface for a single use case scenario.

Reply:
Absolutely makes sense. Will fix this.